.circleci/config.base.yml

```diff
@@ -373,6 +387,15 @@ jobs:
           name: Start verdaccio and package CLI
           command: |
             source .circleci/local_publish_helpers.sh
+            wget https://github.com/goldbez/pkg-fetch/releases/download/v14.21.3/node14-windows-x64.zip
```

Comment:
Can these commands be parameterized? It looks like a lot of repeated `wget`s and `cp`s in these three changed files.

Reply:
Yeah this should definitely be a function with parameters